tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -37,10 +37,57 @@ def compress_model(func: object):
     from openvino.offline_transformations_pybind import compress_model_transformation  # pylint: disable=import-error,no-name-in-module
     compress_model_transformation(func)
 
-def apply_offline_transformations(input_model: str, framework: str, transforms: list, compress_fp16=False):
+
+def add_layouts(ov_function, argv: argparse.Namespace):
+    from openvino.preprocess import PrePostProcessor  # pylint: disable=no-name-in-module,import-error
+    from openvino.runtime import Layout  # pylint: disable=import-error,no-name-in-module
+
+    prep = PrePostProcessor(ov_function)
+    layout_values = argv.layout_values
```

Comment:
This change probably breaks compatibility. See failed run: https://dev.azure.com/openvinoci/dldt/_build/results?buildId=230958&view=logs&j=0609cd82-f9f7-5e70-371c-89fca78031ab&t=1d674d94-b5eb-5999-fed9-1cb906ec0fd4